apm-agent-plugins/apm-jms-plugin/apm-jms-plugin-base/src/main/java/co/elastic/apm/agent/jms/JmsInstrumentationHelper.java

```diff
@@ -49,6 +49,8 @@
 
 public class JmsInstrumentationHelper {
 
+    private static final JmsMessagePropertyAccessor MSG_ACCESSOR = JmsMessagePropertyAccessor.instance();
```

Comment:
[minor] It's already a singleton, I don't think it makes sense to hold a reference to it